keyrod.gemspec

```diff
@@ -0,0 +1,24 @@
+Gem::Specification.new do |spec|
+  spec.name          = 'keyrod'
+  spec.version       = '0.0.0'
```

Comment:
We usually define version within an application main module as a constant and load it from there. This way the version is available within the application, for example if you want to display it in the application's help.
Take a look at [cloudkeeper/cloudkeeper.gemspec](https://github.com/the-cloudkeeper-project/cloudkeeper/blob/master/cloudkeeper.gemspec) and [cloudkeeper/lib/cloudkeeper/version.rb](https://github.com/the-cloudkeeper-project/cloudkeeper/blob/master/lib/cloudkeeper/version.rb).